userspace/libscap/scap_procs.c

```diff
@@ -1002,7 +1002,20 @@ static int32_t _scap_proc_scan_proc_dir_impl(scap_t* handle, char* procdirname,
 		if(res != SCAP_SUCCESS)
 		{
 			snprintf(error, SCAP_LASTERR_SIZE, "cannot add procs tid = %"PRIu64", parenttid = %"PRIi32", dirname = %s, error=%s", tid, parenttid, procdirname, add_error);
-			break;
+			//
+			// Bug 1897: Agent fails to initialize sinsp_worker: error creating the process list
```

Comment:
Let's not refer to the (commercial) agent in the OSS code unless there's no other way. This issue affects all libsinsp users (sysdig, falco). Also, this is not issue 1987 in the public tracker so it will be confusing to future readers of this code.